interfaces/builtin/basedeclaration.go

```diff
@@ -326,6 +326,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  media-hub:
+    allow-installation:
+      slot-snap-type:
+        - app
+    deny-auto-connection: true
```

Comment:
I'm not sure actually, this has been there since the beginning nevertheless I'll remove it.